packages/v4/patternfly-docs/content/accessibility/chip-group/chip-group.md

```diff
@@ -3,17 +3,58 @@ id: Chip group
 section: components
 ---
 
-A **chip group** represents an attribute that has been assigned one or more values. An OR relationship is implied between values in the group. Chip groups are useful to express complex filters to a data set, for example.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-Keyboard users will be able to press **Tab** to navigate through all interactive elements of the chip group (this will include close buttons, any overflow chips with tooltips,  view more buttons, etc). The screen reader experience, likewise, will mostly be based on native interaction expectations.
+## Accessibility
 
-Bear in mind that the `aria-labelledby` attribute on the close button is based on the ID of the chip itself. So, if you are **changing the id of the chip**, you will need to update the `aria-labelledby` on the chip.
+To implement an accessible PatternFly **chip group** component:
 
-<br/>
-In general, the chip group component already has accessibility built in. However, if you’d like to customize it, edit the following:
+- Ensure any interactive elements - such as close buttons or overflow chips - within a chip group can be navigated to and interacted with via keyboard and other assistive technologies such as a screen reader
+- Ensure the entire chip group category name can receive focus, not just the chip group's close button, when the category name is truncated
+- Ensure any truncated text within a chip group category name is displayed as a tooltip when the category name is hovered via mouse or has focus
+- Provide an `aria-label` if the chip group does not have a visible text category name
+- Follow the [tooltip accessibility documentation](/components/tooltip/accessibility) for truncated content
```

Comment:
I think this note may be overkill here since we implement this on behalf of the consumer, right?
It's worth leaving a note about it in the testing section I think, but I like keeping these bullets at the top limited to the considerations the consumer is expected to make when implmeneting a Chip

Reply:
That's a good point. Do you think all 4 of these items could be removed from the first section? The aria-label one might have a case to be left in, but since there is a testing item for it it could be a bit redundant

Reply:
I think we can leave the aria-label note